doc/source/main_install.rst

```diff
@@ -130,28 +116,23 @@ Installing BuildStream
 
 Installing from PyPI
 ~~~~~~~~~~~~~~~~~~~~
-Once you have the base system dependencies, you can install the BuildStream
-python package as a regular user.
 
-To install from PyPI, you will additionally require:
-
- - pip for python3 (only required for setup)
- - Python 3 development libraries and headers
-
-
-For the latest dev snapshot of BuildStream 2, simply run the following command::
+For the latest pre-release of BuildStream 2, including the necessary Python
+dependencies and BuildBox tools, run the following command::
 
     pip3 install --user --pre BuildStream
 
-This will install latest dev snapshot of BuildStream and its pure python
-dependencies into your user's homedir in ``~/.local``.
+This will install BuildStream and its dependencies into your user's homedir in
+``~/.local``.  Pip will use binary "wheel" packages from PyPI where these are
```

Comment:
Nit:

> Pip will use binary "wheel" packages from PyPI where these are available for your platform.

Would probably read better with *"if these are available for your platform"* (whereas *"where"* would be more suitable if *"for your platform"* was removed).

Reply:
Removed in new version